tests/test-sessions/test-mongodb-sessions/pom.xml

```diff
@@ -95,15 +96,27 @@
       <artifactId>jetty-test-helper</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.testcontainers</groupId>
+      <artifactId>testcontainers</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.testcontainers</groupId>
+      <artifactId>junit-jupiter</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-simple</artifactId>
+      <scope>test</scope>
+    </dependency>
   </dependencies>
   <profiles>
     <profile>
-      <id>mongodb</id>
+      <id>ci</id>
```

Comment:
I'd still call the profile "mongodb". It just happens to be being used in a ci environment, but can also be run locally.

Reply:
what about a system property such `-Dmongodb.enabled=true` otherwise we will have a lot of profile here https://github.com/eclipse/jetty.project/blob/0ab2b42055546b8063f83214d5eaf084efdf0aaf/Jenkinsfile#L14 such `-Pmongo -Pinfinispan -Pmemcache -Phazelcast etc` which is a bit complicated to read.
What is the goal exactly of having a profile named `mongodb`?

Reply:
OK, well what about a profile name that is more related to what it is actually doing: something like remote-session-tests or something?

Reply:
Sounds good.
But I will definitely bookmark this comment to be sure I have when I will hear ` grhhh this parameter remote-session-tests is too long when I use the command line 😜

Reply:
Well with the activation test you added to the profile, you never really have to specify anything on the command line :)  Unless, of course, you're using windows .... is there any activation test that is generic enough that would work on most windows environments?

Reply:
I removed the file activation otherwise it is running all the time on linux/osx as long as you have docker and even if I'm happy with that not sure everyone will be? :) 😄